corehq/apps/case_importer/tasks.py

```diff
@@ -279,3 +283,10 @@ def _submit_caseblocks(domain, case_type, caseblocks):
         'errors': errors.as_dict(),
         'num_chunks': num_chunks,
     }
+
+
+total_bytes = datadog_gauge_task(
+    'case_importer.files.total_bytes',
```

Comment:
Thanks, this is the kind of thing I knew I must have missed. Barely any data's come in so I'll just change this now. Out of curiosity

- is there any reason to restrict the length of these? I assume `'commcare.case_importer.files.total_bytes'` is fine, but if there's reason for it to be shorter, or there's some better convention to follow, I can use something else.
- There's no way to change the name of a metric already in datadog, right? I'm imagining a case where we'd already submitted a few months of data to this, and then decided we wanted it prefixed with `commcare`, but didn't want to lose continuity.